### ISSUE TITLE
Accessibility - Backoffice - Search filters issues

### ISSUE THREAD

**Issue by rickdoesburg** [umbraco/Umbraco-CMS#21941]
### Which Umbraco version are you using?

17.2

### Bug summary

While doing a simple WCAG Audit of the backoffice I noticed some bugs.

<img width="702" height="288" alt="Image" src="https://github.com/user-attachments/assets/79cc8e02-d2de-4f40-8801-82185eac4205" />

1. The contrast of the filters buttons are insufficient (1.4.3) 
<img width="108" height="50" alt="Image" src="https://github.com/user-attachments/assets/9252217b-28ef-4e69-ae25-15ac6a8fde13" />  

2. The filters buttons do not communicate their state to screen readers. They do receive an `active` class for styling purposes but should have an `aria-pressed` state for example. 
3. The buttons are not linked to the content they're displaying either using `aria-controls` or making the buttons container a `tablist`, with `tabpanels` and `tabs`.
4. Switching between the buttons doesn't announce the new content being displayed.

### Specifics

Backoffice search

### Steps to reproduce

Press the loop icon in the top bar in the backoffice

### Expected result / actual result

An accessible way of searching in Umbraco using buttons with sufficient contrast, announced content changes and clear information about the active search filter.

**Comment by corevibe555** [comment]
@leekelleher Would it be OK if I work on this?
